### ISSUE TITLE
Allow setting runtimeClassName

### ISSUE THREAD

**Issue by murata-yu** [NVIDIA/dcgm-exporter#102]
Hi, there.

In k3s containerd, set runtimeClass to pod without setting default-runtime to use GPU.
We can't set this in helm chart values.
Do you have any plans to change it so that we can set the runtimeClass?

**Comment by nikkon-dev** [comment]
@murata-yu,

If I got your question right, you should address this to the [nvidia-container-toolkit](https://github.com/NVIDIA/nvidia-container-toolkit). The dcgm-exporter does not have control over the runtime - it just needs to see GPUs inside the container, and the way those are visible does not matter.

**Comment by murata-yu** [comment]
OK, thanks.

**Comment by retsamedoc** [comment]
Er, how was this fixed?

**Comment by nvvfedorov** [comment]
Please confirm that the issue still is not resolved.

**Comment by nvvfedorov** [comment]
@retsamedoc , Please confirm. Do you want to be able to specify runtime in the helm chart?

**Comment by retsamedoc** [comment]
Yes. I would like to specify the runtime in the helm chart using the values.yaml similar to the other charts used for GPU operator.

**Comment by nvvfedorov** [comment]
@retsamedoc , As I see, today you can specify the runtimeClass when you install the dcgm-exporter with the helm: https://github.com/NVIDIA/dcgm-exporter/blob/bb492acd6b3dbf1a2637b86f25312f075820d2ec/deployment/templates/daemonset.yaml#L48.

**Comment by nvvfedorov** [comment]
Done: https://github.com/NVIDIA/dcgm-exporter/pull/291
